front-end/src/Pages/Client/Client-Product/innerPage/PlusMinus.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  AiOutlinePlusCircle,
+  AiOutlineMinusCircle,
+} from 'react-icons/ai';
+import { shoppingListAction } from '../../../../Redux/action/shoppingListAction';
+
+const PlusMinus = ({ index, id, name, photo, price, sellingQuantity, setSellingQuantity }) => {
+  const [disable, setDisable] = useState(false);
+  const dispatch = useDispatch();
+  const shoppingList = useSelector(state => state.shoppingListReducer.data);
+
+  const changeReduxShoppingList = (operator) => {
+    if (shoppingList.length > 0) {
+      if (shoppingList.some((product) => product.id === id)) {
+        return shoppingList.map((product) => {
+          if (product.id === id) {
+            return {
+              id,
+              name,
+              price,
+              photo,
+              sellingQnt: eval(`${product.sellingQnt} ${operator} ${1}`),
```

Comment:
Não seria melhor passar o valor da alteração ao invés do operador?? EX:
```suggestion
              sellingQnt: product.sellingQnt} + operator,
```